README.md

```diff
@@ -133,15 +133,15 @@ of the API. If you do not use this option you will be defaulted to the oldest st
 client := goshopify.NewClient(app, "shopname", "", goshopify.WithVersion("2019-04"))
 ```
 
-#### WithRetry
+#### WithRetry, err
```

Comment:
This line can be reverted

Reply:
Good catch, that should have gone in the line above, for the call to Newclient().
Fixed.